tests/cypress/page-object/site.home.page.ts

```diff
@@ -32,9 +38,25 @@ class SiteHomePage extends BasePage {
             .rightclick()
             .should('have.class', 'context-menu-open')
 
-        this.getIframeBody().contains(this.htmlElements.span, this.elements.textEdit).click()
+        this.getIframeBody().find(this.elements.editSite).click()
```

Comment:
May I ask why we are changing this one? (Is it causing flakiness?)

Reply:
it was changed to accommodate a case where the UI isn't in English